index.html

```diff
@@ -97,7 +97,12 @@ <h2>Security and Privacy Considerations</h2>
           The user visits the website or an <a>associated origin</a>.
         </li>
         <li>
-          The cookies expire or are deleted on the website or an <a>associated origin</a>.
+          If there is a <a>cookie name filter</a> that matches the name of the cookie
```

Comment:
Can you reword this to specify the cookie must be on the website or associated origin?

This is hard to cram into a sentence, but maybe something like:
"If there is a cookie name filter that matches the name of a cookie on the website or associated origin and that cookie's value changed."